frontend/packages/dev-console/integration-tests/features/monitoring/filter-the-alerts.feature

```diff
@@ -0,0 +1,73 @@
+@monitoring
+Feature: Filter for Alert state and Severity
+	User should be able to filter out alerts as per Alert State and Severity
+
+
+    Background:
+        Given user is at developer perspective
+        And user is at Add page 
+        And user has created or selected namespace "aut-monitoring-alerts"
+        And user is at Monitoring page
+
+
+    @smoke
+    Scenario: Filters dropdown on the Alerts page
+        Given user is on the Alerts tab
```

Comment:
Shouldn't we update here `Given` with need / steps to create alert rules so that below all scenarios would work as described